marine/marine.py

```diff
@@ -238,26 +248,26 @@ def __del__(self):
         if getattr(self, "_marine", None):
             self._marine.destroy_marine()
 
-    def _expand_macros(
-        self, fields: List[str], macros: Optional[Dict[str, List[str]]]
+    def _expand_field_templates(
+        self, fields: List[str], field_templates: Optional[Dict[str, List[str]]]
```

Comment:
Maybe also need to run Black here? Or does it place multiple parameters in the same line?

Reply:
It is blacked. It does place the parameters in the same line. 
Same goes for the other formatting comment.